frontend/src/queries.js

```diff
@@ -89,7 +89,14 @@ export const fetchTournaments = async () => {
     headers: { "Content-Type": "application/json" },
     credentials: "same-origin"
   });
-  return await response.json();
+  let tournaments = await response.json();
```

Comment:
if the API fails with error then response.json() will give a object `{"message":"error message"}` like this so will need to handle that here